float64 posX_N
float64 posY_N
float64 distance_pre
int64   detected